usage-elements.py

```diff
@@ -139,152 +118,165 @@
-                ])
-            ])
-        ]),
-
-    ])
-])
+            ],
+        ),
+        html.Div(
+            className="four columns",
+            children=[
+                dcc.Tabs(
+                    id="tabs",
+                    children=[
+                        dcc.Tab(
+                            label="Control Panel",
+                            children=[
+                                drc.NamedDropdown(
+                                    name="Layout",
+                                    id="dropdown-layout",
+                                    options=drc.DropdownOptionsList(
+                                        "random",
+                                        "grid",
+                                        "circle",
+                                        "concentric",
+                                        "breadthfirst",
+                                        "cose",
+                                    ),
+                                    value="grid",
+                                    clearable=False,
+                                ),
+                                drc.NamedRadioItems(
+                                    name="Expand",
+                                    id="radio-expand",
+                                    options=drc.DropdownOptionsList(
+                                        "followers", "following"
+                                    ),
+                                    value="followers",
+                                ),
+                            ],
+                        ),
+                        dcc.Tab(
+                            label="JSON",
+                            children=[
+                                html.Div(
+                                    style=styles["tab"],
+                                    children=[
+                                        html.P("Node Object JSON:"),
+                                        html.Pre(
+                                            id="tap-node-json-output",
+                                            style=styles["json-output"],
+                                        ),
+                                        html.P("Edge Object JSON:"),
+                                        html.Pre(
+                                            id="tap-edge-json-output",
+                                            style=styles["json-output"],
+                                        ),
+                                    ],
+                                )
+                            ],
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    ]
+)
 
 
 # ############################## CALLBACKS ####################################
-@app.callback(Output('tap-node-json-output', 'children'),
-              [Input('cytoscape', 'tapNode')])
+@app.callback(
+    Output("tap-node-json-output", "children"), [Input("cytoscape", "tapNode")]
```

Comment:
```suggestion
    Output("tap-node-json-output", "children"), Input("cytoscape", "tapNode")
```
We can modernize all the callback definitions